docs/plugins/spectate.rst

```diff
@@ -11,7 +11,30 @@ Usage
 
 ::
 
-    enable spectate
+    spectate
```

Comment:
if there is now a command, the `dfhack-tool` directive needs to be updated.

Reply:
I wouldn't claim there is a command. It's more like a self-report the plugin has built-in. I imagine I'll see a related option when I do my documenting documentation reading

Reply:
I think this section is correct, or at least doesn't have this problem anymore